spl-tokens/app/app/README.md

```
@@ -0,0 +1,25 @@
+### Yarn
+
+```bash
+yarn install
+```
+
+```bash
+yarn start
+```
+
+## 💚 Open Source
+
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 3

(MD009, no-trailing-spaces)